source/guides/install-deploy-upgrade-scale.rst

```diff
@@ -31,10 +31,11 @@ Mattermost Server
    Installing Mattermost Omnibus </install/installing-mattermost-omnibus.rst>
    Installing on Kubernetes </install/install-kubernetes.rst>
    Installing on Debian Buster </install/install-debian.rst>
-   Installing on RHEL 7 </install/install-rhel-7.rst>
    Installing on RHEL 8 </install/install-rhel-8.rst>
+   Installing on RHEL 7 </install/install-rhel-7.rst>
```

Comment:
I re-ordered the RHEL install topics from newest to oldest to match the Ubuntu install topics